bpf/README.md

```diff
@@ -68,4 +75,21 @@ BPF programs and user space. BPF programs can only access maps using BPF helpers
 Example:
 
 * BPF: https://github.com/gravitational/teleport/blob/2fd8f75e38eebf5c6826ef594433e5165c3bfbe1/bpf/enhancedrecording/command.bpf.c#L18
-* User space: https://github.com/gravitational/teleport/blob/2fd8f75e38eebf5c6826ef594433e5165c3bfbe1/lib/bpf/common_linux.go#L40-L51
\ No newline at end of file
+* User space: https://github.com/gravitational/teleport/blob/2fd8f75e38eebf5c6826ef594433e5165c3bfbe1/lib/bpf/common_linux.go#L40-L51
+
+## BPF in Teleport
+
+Teleport uses BPF to implement enhanced session recording and restricted networking. Both features work only on Linux with
+the kernel 5.8+. Enhanced session recording records all:
+* exec family system calls
+* open family system calls
+* network connections
+
+All events are recorded in the audit log. https://goteleport.com/docs/server-access/guides/bpf-session-recording/
+
+Restricted networking allows you to restrict network access for users.
+It's implemented by using LSM hooks and BPF programs.
+https://goteleport.com/docs/server-access/guides/restricted-session/
+On ubuntu systems LSM hooks are not enabled in some version.
```

Comment:
```suggestion
On ubuntu systems LSM hooks are not enabled in some versions.
```

I think?

Could we be more specific? (Fine if not.)

Reply:
I think this is deterministic (each version that I personally checked had this disabled), but I think that would be a bold move to claim that LSM are always disabled based on that limited test.